src/server.js

```diff
@@ -28,7 +57,7 @@ const plugin = (app: App, {store, env = [], deps = {}}: Object = {}) => {
     // collect data once at startup to at least have some data in case of a crash
     // $FlowFixMe
     data.version = require(`${__dirname}/../package.json`).version; // eslint-disable-line import/no-dynamic-require
-    data.server = collectDependencyData(app);
+    data.server = [collectDependencyData(app)];
```

Comment:
any reason `data.server` has to be an array type ?

Reply:
It's a super contrived scenario but it's technically possible to spawn multiple servers or cluster workers and each have a different DI graph based on arbitrary conditionals.

I just made it an array after a discussion w/ Ryan about making it less prone to schema breakage down the road.